tecken/storage.py

```diff
@@ -60,7 +60,7 @@ class StorageBucket:
         "test-s3": "s3.example.com",
     }
 
-    def __init__(self, url, try_symbols=False, file_prefix=""):
+    def __init__(self, url, try_symbols=False, file_prefix=settings.SYMBOL_FILE_PREFIX):
```

Comment:
The prefix is currently `v1`, and as far as I'm aware it has never been changed. The idea seems to be that we can change the structure of the storage and then use `v2` as a prefix for the new files, but I don't see this happening anytime soon. I plan to simply stick with the current prefix in GCP.

Instead of introducing the `SUMBOL_FILE_PREFIX` thing, we could have simply appended `/v1` to all bucket URLs. A comment in the tests states "The idea with `settings.SYMBOL_FILE_PREFIX` is to make it easier to specify the `settings.SYMBOL_URLS`." I'm not really convinced not having to suffix the URLs with `/v1` justifies the complexity of the code and the tests implementing the prefix setting, but at least I reduced the complexity a bit in this PR